File: Most/ConsoleApp1/ConsoleApp1/KPlyta.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace ConsoleApp1
{
		class KPlyta : Plyta
		{
			public KPlyta(Tlok type, String name, PlytaInterface plytkaInterface) : base(type, name, plytkaInterface)
			{
			}

			public override void Akcja()
			{
				plytkaInterface.Akcja();
			}

		}
}

File: Design-patterns/Most/Bridge/KPlyta.ts
import { Tlok } from './PlytaInterface';
import { Plyta } from './Plyta';
import { PlytaInterface } from './PlytaInterface';

export class KPlyta extends Plyta
{
    constructor(type: Tlok, name: string, plytaInterface: PlytaInterface)
    {
        super(type, name, plytaInterface)

    }
}